README.md

```diff
@@ -285,6 +285,14 @@ environment variable. We first document the most relevant and most commonly used
   `TERM` environment variable is excluded by default to [speed up the test
   harness](https://github.com/rust-lang/miri/issues/1702). This has no effect unless
   `-Zmiri-disable-isolation` is also set.
+* `-Zmiri-extern-so-file=<path to a shared object file>` is an experimental flag for providing support
```

Comment:
Given how very experimental this feature is, please move it down to the other section for more advanced and more unsafe flags.